packages/notionable/src/base-trade/sagas/vault-risk.ts

```diff
@@ -97,14 +103,40 @@ export function postVaultAccountRisk(
             ?.interestRate
         );
 
+        const vaultCapacity =
```

Comment:
Spacing in this code looks really odd

Reply:
I copy and pasted this from the use-vault-capacity hook